StaticAnalyzer/tools/strings.py

```diff
@@ -0,0 +1,21 @@
+"""
+Strings-module to be undependend from operating systems
+Taken from: http://stackoverflow.com/a/17197027
+"""
+import string
+
+def strings(filename, min=4):
+    """Print out all connected series of readable chars longer than min."""
+    with open(filename, "rb") as f:
+        result = ""
+        for c in f.read():
+            if c in string.printable:
+                result += c
+                continue
+            if len(result) >= min:
```

Comment:
Hmm, wrapping each result in ticks shows why these are shown:
```
'F
  Fe'
'2AF
'
'TopBorder'
'FarmKingHardCurrencyPackageX5'
'FRh0Fk'
'yVz`zjztz~z'
'z
{x{'
```

Newline seems to trick us here.
> string.printable
    String of characters which are considered printable. This is a combination of digits, letters, punctuation, and whitespace.

So:
```'0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~ \t\n\r\x0b\x0c'```
is the original, I would use:
```'0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ!"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~ '```

Result:
```
'0(0bpvp'
'FC`+h'
'9F*bhb"F(F'
'DiamondDiggerCollaborationSlots1Id5_Mobile'
'b0F!F'
'fakepurchase%d%d'
'v wt8'
'CandyMobileTencentColorBombIntroOffer'
'uAcid'
'FarmKingKakaoCampaignSoftCurrencyP1'
'wHxbx~x'
'FBF F$h9F'
'Self'
'tDt\txt'
```